x/liquidstakeibc/keeper/unbonding.go

```diff
@@ -67,11 +68,16 @@ func (k *Keeper) IncreaseUndelegatingAmountForEpoch(
 			IbcSequenceId: "",
 			State:         types.Unbonding_UNBONDING_PENDING,
 		}
+
+		telemetry.ModuleSetGauge(types.ModuleName, float32(epochNumber), chainID, "unbonding_epoch")
 	} else {
 		unbonding.UnbondAmount = unbonding.UnbondAmount.Add(unbondAmount)
 		unbonding.BurnAmount = unbonding.BurnAmount.Add(burnAmount)
 	}
 
+	telemetry.ModuleSetGauge(types.ModuleName, float32(unbonding.UnbondAmount.Amount.Int64()), chainID, "unbond_amount")
```

Comment:
check if amount is int64 convertable.. orelse this will panic

Reply:
`sdk.Int` is convertible to `int64` by default, is there a case that would panic ?